pom.xml

```diff
@@ -1648,7 +1648,7 @@
               <dependency>
                 <groupId>sun.jdk</groupId>
                 <artifactId>tools</artifactId>
-                <version>1.6</version>
+                <version>${required.jvm}</version>
```

Comment:
It's set to 1.7 in parent pom, and (just recently) overridden to 1.8 in
server pom.

On 30 September 2015 at 08:24, Alex Eng <notifications@github.com> wrote:

> In pom.xml
> <https://github.com/zanata/zanata-server/pull/960#discussion_r40738523>:
>
> > @@ -1648,7 +1648,7 @@
> >                <dependency>
> >                  <groupId>sun.jdk</groupId>
> >                  <artifactId>tools</artifactId>
> > -                <version>1.6</version>
> > +                <version>${required.jvm}</version>
>
> Where is the {required.jvm} defined?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-server/pull/960/files#r40738523>.
>



-- 
Sean Flanigan

Principal Software Engineer
Globalisation Tools Engineering
Red Hat